Update default component heights (#984)
sumup-oss/circuit-ui: .changeset/shy-games-poke.md

---
'@sumup/circuit-ui': major
---

> Spinner component has been updated to accept new size values. The size prop has been added with new size values **_byte_**, **_kilo_** and **_giga_**.

Reviewer: ```suggestion
Added a `size` prop to the Spinner component. The possible values are `byte`, `kilo` (default), and `giga`.
```
